core.go

```diff
@@ -286,18 +327,19 @@ func (l *LevelEnabler) Enabled(lvl zapcore.Level) bool {
 // follow same logic with sentry-go to filter unnecessary frames
 // ref:
 // https://github.com/getsentry/sentry-go/blob/362a80dcc41f9ad11c8df556104db3efa27a419e/stacktrace.go#L256-L280
-func filterFrames(frames []sentry.Frame) []sentry.Frame {
+func filterFrames(frames []sentry.Frame, filters FrameFilters) []sentry.Frame {
```

Comment:
I'd refactor this as a method of the core so that we don't have to pass the filters here.

Reply:
And it would be great to have a test for this. Just a bonus :)